console/src/components/Services/Data/TableBrowseRows/EditItem.js

```diff
@@ -77,6 +77,31 @@ class EditItem extends Component {
         defaultNode: null,
       };
 
+      const onChange = (e, val) => {
+        const textValue = typeof val === 'string' ? val : e.target.value;
+
+        const radioToSelectWhenEmpty = prevValue
+          ? refs[colName].defaultNode
+          : refs[colName].nullNode;
+
+        refs[colName].insertRadioNode.checked = !!textValue.length;
+        radioToSelectWhenEmpty.checked = !textValue.length;
+      };
+      const onFocus = e => {
+        const textValue = e.target.value;
+        if (
+          textValue === undefined ||
+          textValue === null ||
+          textValue.length === 0
+        ) {
+          const radioToSelectWhenEmpty = prevValue
+            ? refs[colName].defaultNode
+            : refs[colName].nullNode;
+
+          refs[colName].insertRadioNode.checked = false;
```

Comment:
I think it would be nice if we can plan for removing direct DOM controls and using local state instead.
@beerose thoughts?

Reply:
Yeah, totally agree, it would be. However, this is a pretty complex and big portion of code and refactoring it may take some time. So I'd leave it for later.